.github/workflows/ci.yaml

```diff
@@ -11,7 +11,7 @@ on:
 env:
   CARGO_TERM_COLOR: always
   # Run a decent number of samples for our randomized
-  ZQ_TEST_SAMPLES: 100
+  ZQ_TEST_SAMPLES: 50
```

Comment:
I'd argue against this. The fact that we managed to break the tests suggests we didn't have enough samples, so reducing them further makes me worry. What's the motivation for this?

Reply:
The CI struggles to run the tests/dies and it also takes a long time. This is a significant burden. I would argue that whatever is needed in the short term to make reduce overhead on PRs to a managable level is worth it